src/commands/importExportEntities/entities/utils/reference.ts

```diff
@@ -0,0 +1,72 @@
+import { notNullOrUndefined } from "../../../../utils/typeguards.js";
+import { Transformable } from "../../../../utils/types.js";
+import { ImportContext, SimplifyContext } from "../../entityDefinition.js";
+
+type Params = Readonly<{
+  newId: string | undefined;
+  oldId: string;
+  entityName: string;
+}>;
+
+export const createReference = (params: Params) =>
+  params.newId ? { id: params.newId } : { external_id: `non-existent-${params.entityName}-${params.oldId}` };
+
+/**
+ * extracts top level values from import context. `string:string` pairs are left unchanged,
+ * `string:object` are simplified, with `selfId` string replacing the object as value.
+ *
+ * @param context `ImportContext` object
+ * @param keys specifies which root keys to extract from the context, defaults to all keys
+ * @returns flattened map of string:string pairs
+ */
+export const simplifyContext = <T extends keyof ImportContext = keyof ImportContext>(
```

Comment:
Okay this will probably work as both will point to the zero uuid also, however it would be nice to  check, maybe add comment, that it is dealt with :)